packages/sirius-web/frontend/sirius-web-application/src/hooks/ViewerContext.tsx

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import React from 'react';
```

Comment:
I'll also move those files from this folder. to not have generic folders such as "hook", it would just attract different concerns in the same folder

Reply:
Done